test-e2e/app/Tests/SocialNetworks/Facebook.e2e-spec.ts

```diff
@@ -0,0 +1,93 @@
+import { $, browser } from 'protractor';
```

Comment:
$ is never used
in WEbstorm use "CTRL+ALT+O" as a shortcut for removing unused imports

Reply:
Done.